web/styles/antd-default.js

```diff
@@ -0,0 +1,37 @@
+// TODO generate this file using style-definitions somehow
+
+const lodash = require('lodash');
+
+const AntdStaticTokens = {
+  fontFamily:
+    "'Inter', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, 'Noto Sans', sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji'",
+};
+
+const AntdDefaultCustomTokens = {
+  colorLink: '#6544e9',
```

Comment:
Can we use `getComputedStyle` here to look up CSS variable values from Javascript instead of the hardcoded values? Even if we did generate this using style-definitions, it would still be hard-coded values unless we supported a dynamic lookup.

Reply:
`getComputedStyle` only exists as a window function which means runtime only. I set it up with static values for the initial page load. At build it extracts those for the initial style via `StyleProvider`.

Could ignore this for the main page, but the admin will have issues. I can't remember if just a basic `useEffect` triggers it after the CSS is processed or before. I'm using the `Theme` component's inserted style as the trigger, but it doesn't exist in the admin pages, nor the storybook stuff. Could forgo the static values for the antd defaults if there's some trigger to use it with, but the initial load might look off.

There's also a potential way to simplify all this by using a CSS file instead of this js, there would just be a ton to go through and would need to figure out the selector priority:

```
body .css-var-antd-hash.ant-component {
  --antd-component-color-primary: var(--theme-color-primary);
  ...
}
```

Sorry, I kind of started procrastinating doing this.